### PR TITLE
Wotta - plot formatting updates and create MCMC_output.py

### PR DESCRIPTION
**Pull request by cwotta**
(1) A small update to mcmc.py that affects plot title positioning and x-axis
(2) Small Python3 compatibility fixes (mostly *.pkl file-related)
(3) Creation of 2 files:
(3a) MCMC_output.py -- class that handles output *.pkl/*.hd5 files, for grabbing desired data on a large scale.  Documentation is at the top for easier finding (will likely refer to it each time), not within each function.
(3b) README_python_get_pdf_cdf.py -- example of how to use MCMC_output.py

### CODE REVIEW

File: docs/LLS_Metallicities-Wotta/README_python_get_pdf_cdf.py [hunk 1]
```diff
@@ -0,0 +1,196 @@
+##Example: How to get walkers or create a PDF and CDF
```
Comment (profxj) [inline docs/LLS_Metallicities-Wotta/README_python_get_pdf_cdf.py]: If this is meant as a tutorial of sorts,
can we make it a Notebook instead?

I should prefer that for stuff within docs/

File: pyigm/metallicity/MCMC_output.py [hunk 2]
```diff
@@ -0,0 +1,849 @@
+import numpy as np
```
Comment (profxj) [inline pyigm/metallicity/MCMC_output.py:9]: Please start this module off with a doc description
of what it is for.  Am not sure myself.

File: pyigm/metallicity/MCMC_output.py [hunk 3]
```diff
@@ -0,0 +1,849 @@
...
+
+
+
+################################################
+################################################
+################################################
+
+
+
+class MCMC_output(object):
```
Comment (profxj) [inline pyigm/metallicity/MCMC_output.py:72]: this looks great!

**Review by profxj — CHANGES_REQUESTED** [review]
Just a few things to fuss with.

Thanks for all of this!

**Comment by cwotta** [comment]
Requested changes have been made.  I have been working on updating the MCMC_output class (a few large changes), but I need to test more, before updating here.

**Comment by profxj** [comment]
roger that, @cwotta 

Merge when you are happy.

**Review by profxj — APPROVED** [review]

**Pull request merged**